capstone/src/test/java/com/google/univiz/scorecard/ScorecardConverterTest.java

```diff
@@ -0,0 +1,52 @@
+package com.google.univiz.scorecard;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.base.Converter;
+import com.google.univiz.CarnegieSizeDegree;
+import com.google.univiz.CollegeData;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class ScorecardConverterTest {
+
+  @Test
+  public void converterTest() throws Exception {
+    ScorecardData.Builder scorecardBuilder = ScorecardData.builder();
+    ScorecardData scorecardData =
+        scorecardBuilder
+            .setId(193900)
```

Comment:
nit - consider making these fields constants s.t. you can reference them when building the test objects and when asserting:

```java
private static final int COLLEGE_ID = 1234;
...
ScorecardData scorecardData =
        scorecardBuilder
            .setId(COLLEGE_ID)
...
assertThat(college.id().id()).isEqualTo(COLLEGE_ID);